content/python/concepts/assert/assert.md

```diff
@@ -0,0 +1,43 @@
+---
+Title: 'assert'
+Description: 'A keyword for confirming the truthiness of a given statement'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Assert'
+  - 'Debugging'
+  - 'Development'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+A keyword for confirming the truthiness of a given statement that is often used to facilitate development, document code and debug.
```

Comment:
```suggestion
A keyword for confirming the truthiness of a given statement that is often used to facilitate development, document code, and debug.
```

Reply:
This punctuation was intentional, the requirement of commas before every item in a list is a common misconception. Previously, I would always add these; however, it's really only required when joining independent clauses.

Reply:
@sonnynomnom can we confirm? Full disclosure, I'm not too familiar with the ins-and-outs of the oxford comma but I assumed it was to be used all the time?

Reply:
Yeah, I just checked again. I think for Codecademy content, the oxford comma is required according to the Curriculum Style Guide created by a bunch of our colleagues:

https://curriculum-documentation.codecademy.com/content-guidelines/editorial-pedagogy/#punctuation